Measurements/Measurements.ipynb

```diff
@@ -572,6 +572,43 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Solution:**\n",
+    "Let ${|E_a⟩, |E_b⟩}$ be a measurement with two outcomes $a$ and $b$, which we identify with the answers, i.e., \"a\" = \"state was |0⟩\" and \"b = state was |+⟩\". Then we define\n",
+    "<li>\n",
+    "<ul>$P(a|0)$ = probability to observe first outcome given that the state was $|0⟩$</ul>\n",
+    "<ul>$P(b|0)$ = probability to observe second outcome given that the state was $|0⟩$</ul>\n",
+    "<ul>$P(a|+)$ = probability to observe first outcome given that the state was $|+⟩$</ul>\n",
+    "<ul>$P(b|+)$ = probability to observe second outcome given that the state was $|+⟩$</ul>\n",
+    "</li>\n",
+    "The task is to maximize the probability to be correct on a single shot experiment, which is the same as to minimize the probability to be wrong (on a single shot). Assuming uniform prior, i.e., $P(+) = P(0) = \\frac{1}{2}$, we get<br>\n",
+    "$P_{correct} = P(0) P(a|0) + P(+) P(b|+) = \\frac{1}{2} (P(a|0) + P(b|+))$\n",
+    "\n",
+    "Assuming a von Neumann measurement of the form\n",
+    "<li>\n",
+    "<ul>$E_a = R_y(2\\alpha) \\begin{bmatrix} 1\\\\0\\end{bmatrix} = \\begin{bmatrix}cos(\\alpha) \\\\sin(\\alpha)\\end{bmatrix}$ and</ul>\n",
+    "<ul>$E_b = R_y(2\\alpha) \\begin{bmatrix} 0\\\\1\\end{bmatrix} = \\begin{bmatrix}-sin(\\alpha) \\\\ cos(\\alpha)\\end{bmatrix}$, we get</ul>\n",
+    "<ul>$P(a|0) = |⟨E_a|0⟩|² = cos²(\\alpha)$,</ul>\n",
+    "<ul>$P(b|+) = |⟨E_b|+⟩|² = \\frac{1}{2} - cos(\\alpha) sin(\\alpha)$, and</ul>\n",
+    "<ul>$P_{correct} = \\frac{1}{2} (\\frac{1}{2} + cos²(\\alpha) - cos(\\alpha) sin(\\alpha))$.</ul>\n",
+    "</li>\n",
+    "Maximizing this for $\\alpha$, we get max $P_{success} = \\frac{1}{2} (1 + \\frac{1}{\\sqrt(2)}) = 0.8535...$, which is attained for alpha = $-π/8$.\n",
+    "This means, $|E_a⟩$ is rotated by $-π/8$ compared to $0⟩$(analog $|E_a⟩$ and $1$). If we rotate the system by $-\\alpha=π/8$ we will get $|E_a⟩=|0⟩$ and $|E_b⟩=|1⟩$ and a measurement in the computational basis will get the correct result with a probability of 85%.\n",
+    "\n",
+    "![Task201_Rotation.png](attachment:Task201_Rotation.png)\n",
```

Comment:
I prefer to keep images in separate files, rather than embedding them in the notebook - it makes editing them and viewing them a lot easier. Also, could you please move the image to `img` subfolder inside `Measurements`, similar to how [Superposition kata](https://github.com/microsoft/QuantumKatas/tree/master/Superposition) handles them? The following suggestion should remove the attachment and work once you move the image to a subfolder.

```suggestion
    "![Task201_Rotation.png](./img/Task201_Rotation.png)\n",
```